components/public-api-server/pkg/apiv1/oidc_test.go

```diff
@@ -713,6 +701,32 @@ func TestOIDCService_SetClientConfigActivation_WithFeatureFlagEnabled(t *testing
 		require.NoError(t, err)
 		require.Equal(t, false, getFirstConfigResponse.Msg.GetConfig().Active)
 	})
+
+	t.Run("deactivates record", func(t *testing.T) {
```

Comment:
Do you need tests for not found?

Reply:
added "record not found" test case. 
thanks for catching!